microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -0,0 +1,1231 @@
+---
```

Comment:
Do we have a systematic way for folks to add to this doc as new apis come out? Might be worthwhile to add a step like this to the api addition one note page.

Reply:
Yeah we have discussed adding to this doc each month during the release so whoever does the release notes also adds to this doc if needed.